src/tests/infraestructure/routes/test-routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+import { TestController } from '../controller/TestController';
+
+const router = Router();
+const controller = new TestController();
+
+router.get('/', (req, res) => controller.getTests(req, res));
+
+router.get('/:id', (req, res) => controller.getTestById(req, res));
+
+router.post('/:id/start', (req, res) => controller.startTest(req, res));
+
+router.get('/:id/questions', (req, res) => controller.getQuestions(req, res));
+
+router.post('/:id/submit', (req, res) => controller.submitTest(req, res));
```

Comment:
The submit endpoint accepts arbitrary JSON for answers. There's no request schema validation here; malformed or malicious payloads can reach the use-case.

**Sugerencia:**
```typescript
Add request-body validation (e.g. Joi / Zod) that verifies answers is an array of allowed shapes and limits sizes (max number of answers). Reject invalid payload with 400. Priority: important (input validation / DoS protection)
```